### PR TITLE
UI: update translations during scons build

### PR DESCRIPTION
**Pull request by adeebshihadeh**

### CODE REVIEW

**Comment by sshane** [comment]
Diff is because we don't include a lot of the source files in the release version. This is why I had to exclude the translation test from pre-commit in release CI

**Comment by adeebshihadeh** [comment]
> Diff is because we don't include a lot of the source files in the release version. This is why I had to exclude the translation test from pre-commit in release CI

Can you fix that? The environments shouldn't be different; we can either add them to the release files or only run the translations for the main UI. I'll wait on this PR until that's in.

**Comment by sshane** [comment]
Oh for some reason I thought we didn't include any UI source files. Yep, I can make sure we include all relevant

File: release/check-dirty.sh [hunk 1]
```diff
@@ -7,5 +7,6 @@ cd $DIR
 if [ ! -z "$(git status --porcelain)" ]; then
   echo "Dirty working tree after build:"
   git status --porcelain
+  git diff
```
Comment (sshane) [inline release/check-dirty.sh:10]:
```suggestion
```

**Pull request merged**